pixl_imaging/src/pixl_imaging/_orthanc.py

```diff
@@ -212,6 +248,6 @@ async def raise_if_pending_jobs(self) -> None:
     def aet(self) -> str:
         return str(config("ORTHANC_RAW_AE_TITLE"))
 
-    async def send_existing_study_to_anon(self, resource_id: str) -> Any:
+    async def send_study_to_anon(self, resource_id: str) -> Any:
```

Comment:
oh actually, we should make this an `"Synchronous": False,` job and wait for the job to complete